lib/cli/blueprints/app/files/package.json

```diff
@@ -2,8 +2,8 @@
   "name": "<%= name %>",
   "version": "0.0.1",
   "scripts": {
-    "start": "index.js",
-    "test": "./node_modules/.bin/denali test"
+    "start": "denali server",
```

Comment:
This would assume the user has denali installed globally, right? Using the `node_modules/.bin/denali` should ensure that we use the local installation I think.